font-nanum.yaml

```
@@ -0,0 +1,35 @@
+package:
+  name: font-nanum
+  version: 20200506
+  epoch: 0
+  description: telugu fonts
+  copyright:
+    - license: OFL-1.1
+  dependencies:
+    provides:
+      - fonts-nanum=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 157803748198151250a3cf7cb03aef844e408e9c98849dd889e191846aacad4b
+      uri: https://archive.ubuntu.com/ubuntu/pool/universe/f/fonts-nanum/fonts-nanum_${{package.version}}.orig.tar.xz
+
+  - runs: |
+      mkdir -p "${{targets.contextdir}}/usr/share/fonts/truetype/nanum"
+      mv ./*.ttf "${{targets.contextdir}}/usr/share/fonts/truetype/nanum/"
+
+update:
+  enabled: false
+
+# test/fonts wont work atm, needs changes
+test:
+  pipeline:
+    - runs: |
+        stat /usr/share/fonts/truetype/nanum/
```

We updated the pipeline to support other types, this should work now: (let's rebase from main)

```suggestion
test:
  pipeline:
    - uses: test/fonts
```